api/templates/model_simple.mustache

```diff
@@ -0,0 +1,391 @@
+// {{classname}}{{#description}} {{{description}}}{{/description}}{{^description}} struct for {{{classname}}}{{/description}}
```

Comment:
This file is copy-pasted from the source code of the generator we use. The only edit I made was to add `yaml:` tags to the templates that generate struct fields.